<javaelement>
	<package>
		javax.naming.directory
	</package>
	<type>
		<name>
			public class AttributeInUseException  extends NamingException
		</name>
		<description>
			This exception is thrown when an operation attempts to add an attribute that already exists. Synchronization and serialization issues that apply to NamingException apply directly here.
		</description>
		<methods>
			<method>
				<name>
					public AttributeInUseException(String explanation)
				</name>
				<description>
					Constructs a new instance of AttributeInUseException with an explanation. All other fields are set to null.
				</description>
				
			</method>
			<method>
				<name>
					public AttributeInUseException()
				</name>
				<description>
					Constructs a new instance of AttributeInUseException. All fields are initialized to null.
				</description>
				
			</method>
			
		</methods>
		
	</type>
	
</javaelement>
